pkg/controller/certificaterequests/venafi/venafi_test.go

```diff
@@ -146,6 +147,10 @@ func TestSign(t *testing.T) {
 		}),
 	)
 
+	tppCRWithCustomFields := gen.CertificateRequestFrom(tppCR, gen.SetCertificateRequestAnnotations(map[string]string{"venafi.cert-manager.io/custom-fields": `[{"name": "cert-manager-test", "value": "test ok"}]`}))
+
+	tppCRWithInvalidCustomFields := gen.CertificateRequestFrom(tppCR, gen.SetCertificateRequestAnnotations(map[string]string{"venafi.cert-manager.io/custom-fields": `[{"name": cert-manager-test}]`}))
```

Comment:
We don't have any test coverage of the `type` field or possible values for it - what if an invalid type is specified?

Reply:
Added a test for it as now our package handles the types